tests/flow/test_effects.py

```
@@ -0,0 +1,855 @@
+    def effects_disabled(self):
+        return not self.effects_enabled()
+
+    def monitor_thread(self):
+        global MONITOR_ATTACHED
+        try:
+            with self.replica.monitor() as m:
+                MONITOR_ATTACHED = True
+                for cmd in m.listen():
+                    if 'GRAPH.EFFECT' in cmd['command'] or 'GRAPH.QUERY' in cmd['command']:
+                        self.monitor.append(cmd)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)